wiki/Geki/en.md

```diff
@@ -9,6 +8,46 @@ tags:
 
 *See also: [Katu](/wiki/Katu)*
 
-**Geki (激)**, or *Elite Beat!*, is a [scoring](/wiki/Score) term used when a player has completed a comboset with the highest possible [accuracy](/wiki/Gameplay/Accuracy) on every note. This idea originates from the Nintendo DS game Elite Beat Agents in which the [osu!standard](/wiki/Game_mode/osu!) [game mode](/wiki/Game_mode) is based on.
+**Geki (激)**, or *Elite Beat!*, is a [scoring](/wiki/Score) term used when a player has completed a comboset with the highest possible [accuracy](/wiki/Gameplay/Accuracy) on every note.
 
-<!-- TODO: Add links-->
+By obtaining a Geki, health bar gain bonus obtained is stronger than just ending a comboset with a non-Geki 300s.
+
+This idea originates from the Nintendo DS game Elite Beat Agents in which the [osu!standard](/wiki/Game_mode/osu!) [game mode](/wiki/Game_mode) is based on.
+
+## Screenshots
+
+In here provided the screenshots of obtaining a Geki/Elite Beat from Elite Beat Agents, Ouendan 2, and osu! itself.
+
+![](img/eba-bornlove-300g.jpg) ![](img/oto-sambomaster-300g.jpg)
+
+![](img/osu-lonelest-300g.jpg)
+
+## On other game modes
+
+### osu!taiko
+
+Geki is only shown on result screen to show all big note hitting succession.
```

Comment:
Doesn't it also show over the approach circle when you hit a big note during game play? Maybe I'm reading this wrong.

Reply:
Sorry for the late reply, apparently it's `hit300k` that being used for "Both Side Big Note" hit.